### PR TITLE
[#1260] Group previews now have the same height.

### PR DESCRIPTION
**Pull request by mauriel133**
Also: [#1260] The entire group image is now displayed, instead of a zoomed in version of it.

### CODE REVIEW

**Comment by pkdash** [comment]
@hyi Can you review the small PR?

File: hs_core/templates/pages/collaborate.html [hunk 1]
```diff
@@ -339,40 +339,18 @@ <h4 class="modal-title" id="myModalLabel">Create New Group</h4>
...
-            }
-            else if (current == "md" || current == "lg") {
-                delta = 3;
-            }
-
             var containers = $('.group-thumbnails').find('div.group-container');
 
+            var maxHeight = 0;
             // Fix height of containers for each row. One row every delta items.
-            for (var i = 0; i < containers.length - delta; i += delta) {
```
Comment (hyi) [inline hs_core/templates/pages/collaborate.html]: @mauriel133 The comment above, particularly on ```one row every delta items``` does not apply any more due to your changes.

File: theme/static/css/hydroshare_core.css [hunk 2]
```diff
@@ -3404,6 +3404,7 @@ div.btn-update-picture:hover {
 
 #user-profile-tabs {
     margin-top: 20px;
+	margin-bottom: 60px;
```
Comment (hyi) [inline theme/static/css/hydroshare_core.css]: @mauriel133 appears to have some tab vs space misalignment in style definition above. Just something to check for to make sure they are aligned properly for readability.

File: theme/static/css/hydroshare_core.css [hunk 3]
```diff
@@ -3475,6 +3476,11 @@ button.no-style:focus {
     line-height: 1.42857143;
     background-color: #fff;
     border: 1px solid #ddd;
+	height: 100%;
+}
```
Comment (hyi) [inline theme/static/css/hydroshare_core.css]: @mauriel133 Tab vs space misalignment here?

File: theme/static/css/hydroshare_core.css [hunk 4]
```diff
@@ -3568,9 +3578,11 @@ button.no-style:focus {
     height: 80px;
     text-align: center;
     width: 80px;
-    background-size: cover;
+    background-size: contain;
+	background-repeat: no-repeat;
     background-position: center center;
     display: inline-block;
+	background-color: #093e54;
```
Comment (hyi) [inline theme/static/css/hydroshare_core.css]: @mauriel133 Tab vs space misalignment here?

File: theme/static/css/hydroshare_core.css [hunk 5]
```diff
@@ -3676,6 +3688,8 @@ button.no-style:focus {
 
 .group-thumbnail-footer {
     padding-top: 10px;
+	position: absolute;
+	bottom: 14px;
```
Comment (hyi) [inline theme/static/css/hydroshare_core.css]: @mauriel133 Tab vs space misalignment here?

**Comment by mauriel133** [comment]
@hyi, I have applied the suggested changes now.

**Pull request merged**